packages/paste-core/components/input-box/src/InputBox.tsx

```diff
@@ -21,7 +21,7 @@ export interface InputBoxProps {
   variant?: Variants;
 }
 
-const InputBox = React.forwardRef<HTMLDivElement, InputBoxProps>(
+const InputBox = React.forwardRef<HTMLDivElement, React.PropsWithChildren<InputBoxProps>>(
```

Comment:
As we're using `React.Props**WithChildren**` InputBox will have already the children prop. No need to define children on the InputBoxProps L11